server/index.js

```diff
@@ -275,6 +275,23 @@ app.post('/api/eventAttendees', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/eventAttendees/:userId', (req, res, next) => {
+  const pastAttendanceEvents = `select
+                                "e"."gameId",
+                                "e"."eventName",
+                                "e"."startTime",
+                                "e"."gameFormat"
```

Comment:
add eventId as well so the user can click on the card sent back. Sending everything * back is acceptable, too